pandas/core/reshape/merge.py

```diff
@@ -47,7 +47,7 @@
 def concat_wrap():
 
     def wrapper(*args, **kwargs):
-        warnings.warn("pandas.tools.merge.concat is deprecated. "
```

Comment:
We will need to keep this one as well (if we thought it was needed). In any case, deprecating the function at the new location `pandas.core.reshape.merge.concat` does not really make sense (no one is using that)